### PR TITLE
Fix instructions for using serverless from the CLI without GitHub

### PR DESCRIPTION
**Pull request by shalabhc**
### Summary & Motivation

The instructions didn't work out of the box. Added a note about the `dagster-cloud` dependency and updated the command to point it at the right directory.

### CODE REVIEW

**Comment by shalabhc** [comment]
Current dependencies on/for this PR:
* master
  * **PR #12161** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/12161" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/12161?utm_source=stack-comment).

File: docs/content/dagster-cloud/deployment/serverless.mdx [hunk 1]
```diff
@@ -76,10 +76,19 @@ dagster-cloud configure
 
 You can also configure the `dagster-cloud` tool noninteractively; see [the CLI docs](/dagster-cloud/developing-testing/dagster-cloud-cli#environment-variables-and-cli-options) for more information.
 
+Add 'dagster-cloud' as a dependency to `my-dagster-project/setup.py`:
```
Comment (erinkcochran87) [inline docs/content/dagster-cloud/deployment/serverless.mdx:79]:
```suggestion
Add `dagster-cloud` as a dependency to `my-dagster-project/setup.py`:
```

🔧 formatting; package/library names should be formatted as code

File: docs/content/dagster-cloud/deployment/serverless.mdx [hunk 2]
```diff
@@ -76,10 +76,19 @@ dagster-cloud configure
...
+
+```
```
Comment (erinkcochran87) [inline docs/content/dagster-cloud/deployment/serverless.mdx:81]:
```suggestion
```python
```

:wrench: code blocks should have a language associated with them so syntax highlighting is applied

File: docs/content/dagster-cloud/deployment/serverless.mdx [hunk 3]
```diff
@@ -76,10 +76,19 @@ dagster-cloud configure
...
+    install_requires=[
+        "dagster",
+        "dagster-cloud",    # add this line
+        "dagster-airbyte",
```
Comment (erinkcochran87) [inline docs/content/dagster-cloud/deployment/serverless.mdx:85]:
```suggestion
install_requires=[
    "dagster",
    "dagster-cloud",    # add this line
    "dagster-airbyte",
```

🔧 In this case, the code should be flush-left. You might need to run `make mdx-format` locally after making this change, or BK could throw a fit.

**Review by erinkcochran87 — COMMENTED** [review]
A few small formatting changes, but otherwise looks good!

File: docs/content/dagster-cloud/deployment/serverless.mdx [hunk 4]
```diff
@@ -76,10 +76,19 @@ dagster-cloud configure
 
 You can also configure the `dagster-cloud` tool noninteractively; see [the CLI docs](/dagster-cloud/developing-testing/dagster-cloud-cli#environment-variables-and-cli-options) for more information.
 
+Add `dagster-cloud` as a dependency to `my-dagster-project/setup.py`:
+
+```python
+  install_requires=[
+      "dagster",
+      "dagster-cloud",    # add this line
+      "dagster-airbyte",
```
Comment (erinkcochran87) [inline docs/content/dagster-cloud/deployment/serverless.mdx:85]: I'm not sure what happened with the GH suggestion I made, but this is still indented a bit too much - could you please fix this before merging? :)

**Review by erinkcochran87 — APPROVED** [review]
:shipit: GH suggestion formatting didn't apply as expected to the code block, but this otherwise looks good! Could you fix that before merging? :)

File: docs/content/dagster-cloud/deployment/serverless.mdx (hunk 4 above)
Comment (shalabhc) [reply docs/content/dagster-cloud/deployment/serverless.mdx:85]: my mistake probably, fixed all the way to the left now.

**Pull request merged**